### ISSUE TITLE
Reduce memory usage by shrinking size of DOOM1.WAD

### ISSUE THREAD

**Issue by FrenkelS** [FrenkelS/Doom8088#4]
Copy and change [GbaWadUtil](https://github.com/doomhack/GbaWadUtil)
* add to WAD `HELP2.LMP`, `STBAR.LMP`, `TITLEPIC.LMP`, `WIMAP0.LMP`
* lower details of pictures by only using even columns and letting odd columns point to the previous column. See [WADptr](https://soulsphere.org/projects/wadptr)
* shave of some unused bytes in [posts](https://doomwiki.org/wiki/Picture_format#Posts), [map textures](https://doomwiki.org/wiki/TEXTURE1_and_TEXTURE2#Map_textures_structure.2C_binary_data) and [map patches](https://doomwiki.org/wiki/TEXTURE1_and_TEXTURE2#Map_patches_structure.2C_binary_data)
* replace texture names by numbers in [sidedefs and sectors](https://github.com/doomhack/GBADoom/issues/26)
* change the palette values in the WAD so it's not necessary to do `(*palette++) >> 2` in `i_system.c`

**Comment by kilrah** [comment]
What would be the lowest storage requirement if e.g. the WAD only contaned E1M1? Just got a neat industrial device using a 386-based microcontroller that runs everything off (currently) 1M of flash EEPROM... would be fun to at least get _something_ going :D
I've ordered chips to hopefully double that but not sure it'll work...

**Comment by FrenkelS** [comment]
Around 226 kB.

Check out miniwad.wad
https://github.com/fragglet/miniwad
https://www.doomworld.com/forum/topic/108721-miniwadwad-a-minimalist-iwad/

**Comment by kilrah** [comment]
Woo this _might_ work, "just" need to wrangle my way into transferring data onto it...

**Comment by FrenkelS** [comment]
See [RP2040 Doom](https://kilograham.github.io/rp2040-doom/flash.html) for more compression tips.

**Comment by fragglet** [comment]
Hi! You might also want to try Squashware Doom which I put together earlier this year:

https://github.com/fragglet/squashware

**Comment by FrenkelS** [comment]
yeah, I put a link to it in the [Jaguar levels](https://github.com/FrenkelS/Doom8088/issues/43) issue.

**Comment by FrenkelS** [comment]
WAD lump merging, Sidedef packing, Graphic squashing and Blockmap stacking are implemented.
See the [wadptr](https://soulsphere.org/projects/wadptr/manual.html) manual for more information.

**Comment by FrenkelS** [comment]
> Hi! You might also want to try Squashware Doom which I put together earlier this year:
> 
> https://github.com/fragglet/squashware

The levels in [My Doom Shrunk In The Wash!](https://www.youtube.com/watch?v=D5x0bpZDkQw) look even more squashed 🤣
